packages/component-submission/server/aggregates/Submission.js

```diff
@@ -77,6 +113,10 @@ class Submission {
       fileStatus: this.filesAreStored() ? 'READY' : 'CHANGING',
       files: this._getFilesWithDownloadLink(),
       teams: this.teams.map(team => team.toJSON()),
+      suggestions: Object.entries(this.suggestions).reduce(
```

Comment:
Is an array going to be the best way of presenting this info? would an object with the field name as the index not be easier to work with on the client?

Reply:
You can't do that with Graphql - the object keys cannot be defined at runtime